sass/styles.scss

```diff
@@ -0,0 +1,178 @@
+//Text-Settings//
+//quick question: can I have a variable with more than one css effect?//
+//I was wondering if I could combine/group codes and set it as a variable// 
+$sans: -apple-system, BlinkMacSystemFont, "Bebas Neue", cursive, sans-serif, Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+$serif: "Cormorant Garamond", serif; 
+$indent: 6vw;
+$text-small: 3vw;
+$text-large: 5vw;
+$undo: none;
+
+
+
+//Basic Document Settings//
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+  
+body {
+    background-color: whitesmoke;
+    color: #242828;
+    font-family: $sans;
+    text-align: left;
+    font-weight: bold;
+    text-transform: uppercase;
+    line-height: 1;
+}
+
+h1, h2, h3 {
+    font-size: $text-large;
+  }
+  
+h4, p, span, ul {
+    font-size: $text-small;
+}
+      
+h4 {
+    font-style: italic;
+}
+
+ul {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+  
+address {
+    font-style: normal;
+}
+
+span {
+    margin-left: $indent;
+    display: block;
+}
+  
+.u-sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute;
+    height: 1px;
+    width: 1px;
+}
+  
+
+
+//Indented Em-dash settings//
+.em-dash-container {
+    display: flex; 
+    justify-content: flex-start; 
+    align-items: center;
+}
+  
+.em-dash {
+    background:black;
+    height: 0.25vw;
+    width: 5vw;
+    margin: 0 1vw 0 0; 
+}
+  
+
+
+//Text Formatting & Styling//
+.indent-event-description { 
+    margin-left: $indent;
+}
+  
+.large-font {
+    font-size: $text-large;
+}
+  
+.small-font {
+    font-size: $text-small;
+}
+  
+
+
+//Adjusting DIV items in Grids//
+main {
+    margin: 1.5%;
+    display: grid;
+    grid-template-columns: repeat(3, 32%);
+    grid-template-rows: repeat(10, 10%);
+    column-gap: 2%;
+    row-gap: 2%
+}
+  
+.item {
+    overflow-wrap: break-word;
+    hyphens: $undo;
+}
+  
+.item:nth-child(1) {
```

Comment:
Your grid works by setting each individual element in its place -- but I think there's a simpler way where you let the grid do some of the work for you.

See [this example](https://glitch.com/edit/#!/isaac-p2-simplified) where I've used `grid-auto-flow: column;` to force the content to flow through the grid by columns (instead of rows, which is the default) and then used a bunch of `span` values on the `grid-row` property to tell each element how much space to take up.

As a result, I get close to the same effect but with:
1. no `grid-column` declarations
2. four of the grid elements (5, 6, 7 and 9) take up the default amount of space and need **no** `grid-column` or `grid-row` rules. 

Let me know if you have questions about the code in this example.